src/switchcraft_advanced/services/intune_service.py

```
@@ -0,0 +1,362 @@
+                startupinfo=startupinfo
+            )
+
+            if process.returncode == 0:
+                logger.info("IntuneWin package created successfully.")
+                return process.stdout
+            else:
+                logger.error(f"IntuneWin creation failed: {process.stderr}")
+                raise RuntimeError(f"Tool exited with code {process.returncode}: {process.stdout}")
+
+        except Exception as e:
+            logger.error(f"Error running IntuneWinAppUtil: {e}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)